server/events/vcs/github_client.go

```diff
@@ -113,6 +139,52 @@ func (g *GithubClient) CreateComment(repo models.Repo, pullNum int, comment stri
 	return nil
 }
 
+func (g *GithubClient) HideOldComments(repo models.Repo, pullNum int) error {
+	// TODO: Paginate issues.
+	comments, _, err := g.client.Issues.ListComments(g.ctx, repo.Owner, repo.Name, pullNum, &github.IssueListCommentsOptions{
+		Sort:        "created",
+		Direction:   "asc",
+		ListOptions: github.ListOptions{},
+	})
+	if err != nil {
+		return err
+	}
+
+	for _, comment := range comments {
+		if comment.User != nil && comment.User.GetLogin() != g.user {
+			continue
+		}
+		// Crude filtering: The comment templates typically include the command name
+		// somewhere in the first line. It's a bit of an assumption, but seems like
+		// a reasonable one, given we've already filtered the comments by the
+		// configured Atlantis user.
+		body := strings.Split(comment.GetBody(), "\n")
+		if !strings.Contains(body[0], models.ApplyCommand.String()) {
```

Comment:
We should look for `"Ran Plan for"` and `**Plan Error**`.

Reply:
I think we need to inject hidden comments in the body. If a plan output is too large, Atlantis seems to split them across comments without a common header.

Reply:
Yes but it's super uncommon for that to happen in my experience. I think it's okay to not handle that edge case

Reply:
How about we just search for "plan"?